stories/ng2-component-lab/checklist.component.exp.ts

```diff
@@ -0,0 +1,91 @@
+import { experimentOn } from '@islavi/ng2-component-lab';
+import {ChecklistItemModel} from "../../src/angular/checklist/models/ChecklistItem";
+import {ChecklistModel} from "../../src/angular/checklist/models/Checklist";
+
+const checklistValuesExample1 = [];
+const checkListExample1: ChecklistModel =  new ChecklistModel(checklistValuesExample1,
+    [new ChecklistItemModel('apple'),
+        new ChecklistItemModel('banana'),
+        new ChecklistItemModel('orange')]);
+
+const checklistValuesExample2 = [];
+const checkListExample2: ChecklistModel =  new ChecklistModel(checklistValuesExample2,
+    [new ChecklistItemModel('apple', false, false, null, 0),
+        new ChecklistItemModel('banana', false, false, null, 1),
+        new ChecklistItemModel('orange', false, false, null, 2)]);
+
+const checklistValuesExample3 = [];
+const checkListExample3: ChecklistModel =  new ChecklistModel(checklistValuesExample3,
+    [new ChecklistItemModel('apple', false, true),
+        new ChecklistItemModel('banana'),
+        new ChecklistItemModel('orange', false, true)]);
+
+const checklistValuesExample4 = [];
+const checkListExample4: ChecklistModel =  new ChecklistModel(checklistValuesExample4,
+    [new ChecklistItemModel('apple', true),
+        new ChecklistItemModel('banana', true),
+        new ChecklistItemModel('orange')]);
+
+const checklistValuesExample5 = [];
+const innerChecklistValues = [];
+const checkListExample5: ChecklistModel =  new ChecklistModel(checklistValuesExample5,
+    [new ChecklistItemModel('apple', false, false,
+            new ChecklistModel(innerChecklistValues,
+                            [new ChecklistItemModel('red'), new ChecklistItemModel('green'), new ChecklistItemModel('yellow')])),
+        new ChecklistItemModel('banana'),
+        new ChecklistItemModel('orange')]);
+
+export default experimentOn('Checklist')
+    .case('Normal Checklist', {
+        showSource: true,
+        context: {
+            checklistModel: checkListExample1,
+            checklistValues: checklistValuesExample1
+        },
+        template: `
+     <span>Selected values: {{checklistValues.toString()}}</span>
+     <sdc-checklist [checklistModel]="checklistModel"></sdc-checklist>
+    `
```

Comment:
Please add to template also how you created the ChecklistModel so users can see how to create this.
Example:
``` 
.case('Normal Checklist', {
        showSource: true,
        context: {
            checklistModel: checkListExample1,
            checklistValues: checklistValuesExample1
        },
        template: `
<h2>Define ChecklistModel</h2>
const checklistValuesExample1 = [];
const checkListExample1: ChecklistModel =  new ChecklistModel(checklistValuesExample1,
    [new ChecklistItemModel('apple'),
       new ChecklistItemModel('banana'),
       new ChecklistItemModel('orange')]);

<h2>Usage</h2>
     <span>Selected values: {{checklistValues.toString()}}</span>
     <sdc-checklist [checklistModel]="checklistModel"></sdc-checklist>
    `
```